event-handlers/src/main/java/no/sikt/nva/nvi/events/cristin/CristinNviReportEventConsumer.java

```diff
@@ -102,4 +113,17 @@ private CandidateDao createAndPersist(CristinNviReport cristinNviReport) {
             return repository.create(candidate, approvals, yearReported);
         }
     }
+
+    private List<CristinDepartmentTransfer> readCristinDepartments() {
+        try {
+            MappingIterator<CristinDepartmentTransfer> iterator =
```

Comment:
Codacy issue: Ensure that resources like this StringReader object are closed after use

Reply:
```
    private List<CristinDepartmentTransfer> readCristinDepartments() {
        try (StringReader reader = new StringReader(CRISTIN_DEPARTMENT_TRANSFERS_STRING)) {
            MappingIterator<CristinDepartmentTransfer> iterator =
                new CsvMapper()
                    .readerFor(CristinDepartmentTransfer.class)
                    .with(CsvSchema.emptySchema().withHeader())
                    .readValues(reader);
            return iterator.readAll();
        } catch (Exception e) {
            throw new RuntimeException(e);
        }
    }
```